src/Paystack.php

```diff
@@ -112,8 +112,9 @@ private function setRequestOptions()
     public function makePaymentRequest( $data = null)
     {
         if ( $data == null ) {
-            $data = [
-                "amount" => intval(request()->amount),
+                $quantity = intval(request()->quantity);
```

Comment:
I think adding multiple quantities is good but it has to be optional. Here, if the user does not offer multiple quantities, what happens?

Reply:
If the user does not offer multiple quantities, $quantity will be 1, it won't have an effect on "amount" => intval(request()->amount)*$quantity,